src/man/sssd.8.xml

```diff
@@ -90,6 +90,10 @@
                         log files are stored in <filename>/var/log/sssd</filename> and
                         there are separate log files for every SSSD service and domain.
                     </para>
+                    <para>
+                        This option is deprecated. It is replaced with
```

Comment:
It is replaced with -> It is replaced by

But here would be nice to hear from @justin-stephenson